### DISCUSSION TITLE
unlock Tx on all bands does not work

### DISCUSSION THREAD

**Discussion by kamiloo78** [egzumer/uv-k5-firmware-custom#256] [Q&A] [answered]
I used Egzumer v 20 and v19 software, unlock Tx on all bands does not work. I tried "Go to [hidden menu
Enter menu F-Lock
Choose option UNLOCK ALL
Repeat steps 2-3 10 times. Do it carefully, if you confirm any other option in the process counter will get zeroed and you will have to repeat that 10 times more."
It doesn't work. Tx Enab is missing from the hidden menu.
Transmission Tx 26.960 -27.410 MHz does not work.Ps.Testing on quansheng uv k5(8)
![IMG_20231213_143050](https://github.com/egzumer/uv-k5-firmware-custom/assets/153738835/ca17f507-e7cb-4e0b-b90b-061cc3cd54e4)
![IMG_20231213_143108](https://github.com/egzumer/uv-k5-firmware-custom/assets/153738835/f68d9ec8-c189-4a45-9898-cb0dfcd3b2ce)
![IMG_20231213_143112](https://github.com/egzumer/uv-k5-firmware-custom/assets/153738835/d91a4b7e-0862-4e84-bbde-93c6c436d8b7)

**Comment by gartnl** [comment, marked as answer]
You are trying to transmit in AM. That is not possible and locked no matter what other settings you have.
The hardware does not support AM TX. Nor does it support TX on CB. Or to be honest any other frequency outside 2/70 HAM bands. 
Please read the strong warning on [https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#tx-on-all-bands](url)
The option is really just a showcase of what the chip can achieve, but not intended for actual use. And if used for testing a dummyload is mandatory.
The frequency in you images would emit a strong 5th harmonic in the middle of the airband. And the 4th harmonic in the broadcast band.

**Reply by kamiloo78** [reply]
Thank you for your answer, now I know everything. Regards

**Comment by ErikS-web** [comment]
The settings are personal and customizable.

Here are some TIPs:
The adjustments I make as standard after flashing a radio...

  1 `Step` - step of the frequency (in kHz). Set it to 12.5 kHz
13 `Demodu` - demodulator mode. Set it to FM 
31 `Mic` - microphone sensitivity (+1.1dB .. +15.1dB). Set it to 15.1
32 `MicBar` - microphone bar that appears while transmitting. Set it to ON
33 `ChDisp` - channel display style (name + frequency or number). Set it to Name + Freq
34 `POnMsg` - Power on message. Set it to Message
35 `BatTxt` - additional battery value on the status bar in % or V(oltage). Set it to Voltage
36 `BackLt` - backlight duration (OFF/ON/configured time). Set it to 2 minutes
56 `AM Fix` - activates the autogain AM fix function. Set it to ON
59 `RxMode` - determine how the upper and lower frequencies are used. Set it to MAIN TX DUAL RX - always broadcasts on the primary station (marked ►), listens to both (DW on screen)

and in the hidden menu

67 `BatCal` - battery calibration, measure the voltage at the back of the radio with a voltmeter and adjust the value in the menu accordingly
68 `BatTyp` - Battery type, 1600 mAh and 2200 mAh battery have very different discharge curve, this is used to calculate battery level percentage. Set it to 1600, or 2200 (mostly UV5RPlus).

**Reply by ErikS-web** [reply]
Today I loaded a new radio with saved Chirp-file (made with the [new Chirp driver](https://github.com/egzumer/uvk5-chirp-driver))

Works perfect. All settings are OK and I only had to do callibration with 67.

**Reply by ErikS-web** [reply]
You mentioned v0.20.
Tip.
Today a [patch v0.20.1 is released. ](https://github.com/egzumer/uv-k5-firmware-custom/releases/tag/v0.20.1)
